src/Runner.Worker/ActionRunner.cs

```diff
@@ -158,8 +158,12 @@ public async Task RunAsync()
             // Setup container stephost for running inside the container.
             if (ExecutionContext.Global.Container != null)
             {
-                // Make sure required container is already created.
-                ArgUtil.NotNullOrEmpty(ExecutionContext.Global.Container.ContainerId, nameof(ExecutionContext.Global.Container.ContainerId));
+                // Make sure required container is already created
+                // Container hooks do not necesseraly set 'ContainerId'
+                if (!FeatureFlagManager.IsContainerHooksEnabled(ExecutionContext.Global.Variables))
```

Comment:
I think the condition should be `ff enabled` + `use container hook`

Reply:
`FeatureFlagManager.IsContainerHooksEnabled` has been changed to check on the FF and the ENV 👍